ksql-api/src/test/java/io/confluent/ksql/api/perf/InsertsStreamRunner.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.perf;
+
+import io.confluent.ksql.api.server.BaseSubscriber;
+import io.confluent.ksql.api.server.BufferedPublisher;
+import io.confluent.ksql.api.server.InsertResult;
+import io.confluent.ksql.api.server.InsertsStreamSubscriber;
+import io.confluent.ksql.api.spi.Endpoints;
+import io.confluent.ksql.api.spi.QueryPublisher;
+import io.vertx.codegen.annotations.Nullable;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.WorkerExecutor;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.parsetools.RecordParser;
+import io.vertx.core.streams.ReadStream;
+import io.vertx.ext.web.codec.BodyCodec;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+public class InsertsStreamRunner extends BasePerfRunner {
+
+  public static void main(String[] args) {
+    new InsertsStreamRunner().go();
+  }
+
+  private SendStream sendStream;
+
+  @Override
+  public void configure() {
+    setNumWarmupRuns(5).setNumRuns(5).setRunMs(10000).setEndpoints(new InsertsStreamEndpoints());
+  }
+
+  @Override
+  public void run(long ms) throws Exception {
+
+    RecordParser parser = RecordParser.newDelimited("\n").handler(row -> {
+      count();
+    });
+
+    sendStream = new SendStream(vertx);
+
+    client.post(8089, "localhost", "/inserts-stream")
+        .as(BodyCodec.pipe(new RunnerUtils.ReceiveStream(parser)))
+        .sendStream(sendStream, ar -> {
+        });
+
+    Thread.sleep(ms);
+
+  }
+
+  @Override
+  protected void endRun() throws Exception {
+    sendStream.pause();
+
+    Thread.sleep(500);
```

Comment:
Why the sleep with a magic number of ms?  Such sleeps in tests are a code smell and often result in intermittent failures.